azure-toolkit-libs/azure-toolkit-storage-lib/src/main/java/com/microsoft/azure/toolkit/lib/storage/service/StorageAccount.java

```diff
@@ -51,6 +56,20 @@ public void delete() {
         }
     }
 
+    @AzureOperation(name = "storage|account.get_connection_string", params = {"this.entity().getName()"}, type = AzureOperation.Type.SERVICE)
+    public String getConnectionString() {
+        // refer https://github.com/Azure/azure-cli/blob/ac3b190d4d/src/azure-cli/azure/cli/command_modules/storage/operations/account.py#L232
+        final String suffix = Azure.az(AzureCloud.class).get().getStorageEndpointSuffix();
```

Comment:
May use `ResourceManagerUtils.getStorageConnectionString(name, key, env)`, refers https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/resourcemanager/azure-resourcemanager-appservice/src/main/java/com/azure/resourcemanager/appservice/implementation/FunctionAppImpl.java#L190